components/CardsLayout.js

```diff
@@ -1,13 +1,49 @@
-import React from 'react';
+import { AiOutlineLeft, AiOutlineRight } from 'react-icons/ai';
+import React, { useRef } from 'react';
+import SVGCardBackground from './SVGCardBackground';
+import { usePosition } from 'hooks/usePosition';
 
 export default function CardsLayout({ children, description }) {
+  const ref = useRef();
+  const { hasItemsOnLeft, hasItemsOnRight, scrollRight, scrollLeft } =
+    usePosition(ref);
   return (
-    <div className="text-center hero-content md:m-auto">
-      <div className="w-lg">
-        <h2 className="pb-20 text-3xl md:text-4xl">{description}</h2>
-        <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 md:gap-x-8 gap-y-5 justify-center align-center content-center">
-          {children}
-        </div>
+    <div className="w-full py-20 relative ">
+      <h2 className="pb-20 text-3xl md:text-4xl z-10 text-center">
+        {description}
+      </h2>
+      <div className="absolute top-0 left-0 h-full w-full overflow-hidden flex justify-center items-center z-0">
+        <SVGCardBackground />
+      </div>
+      <button
+        aria-label="move right"
+        className={`btn btn-accent rounded-sm btn-size absolute right-4 top-1/2  z-50 ${
+          !hasItemsOnRight && 'hidden'
+        }`}
+        // todo: -translate-y-1/2 this also we need to add to make it vertically center
+        // disabled={isPending ? true : false}
+        onClick={scrollRight}
+        type="button"
+      >
+        <AiOutlineRight />
+      </button>
+      <button
+        aria-label="move left"
+        className={`btn btn-accent rounded-sm btn-size absolute left-4 top-1/2 z-50 ${
+          !hasItemsOnLeft && 'hidden'
+        }`}
+        // todo: -translate-y-1/2 this also we need to add to make it vertically center
+        // disabled={isPending ? true : false}
+        onClick={scrollLeft}
+        type="button"
+      >
+        <AiOutlineLeft />
```

Comment:
I like the button use here. The way they appear and operate is very nice.

Reply:
Thanks :)